modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/styles/components/_table.scss

```diff
@@ -0,0 +1,7 @@
+.common-table-checkbox {
+	max-width: 100px;
+}
+
+.common-table-active-row {
+	background-color: #f0f5ff !important;
```

Comment:
There not a bg class color on Dialect?

Reply:
There will be a fix to that, this is not applying in the table and also the color is not right. We need to use #004ad71a. I looked into Dialect colors but I didn't find the same color as above.

Reply:
The closest class that I found was Color Action Primary Active (#004AD7) But is not the same, is slightly darker. Should we change this variable? My concern is affecting other components that uses this class

Reply:
Probably is wrong on Style Book. Recently was changed all Style Book. We need check.